certbot/cert_manager.py

```diff
@@ -165,24 +166,34 @@ def _report_lines(msgs):
     """Format a results report for a category of single-line renewal outcomes"""
     return "  " + "\n  ".join(str(msg) for msg in msgs)
 
-def _report_human_readable(parsed_certs):
+def _report_human_readable(config, parsed_certs):
     """Format a results report for a parsed cert"""
     certinfo = []
+    checker = ocsp.RevocationChecker(config)
     for cert in parsed_certs:
+        if config.certname and cert.lineagename != config.certname:
+            continue
+        if config.domains and not set(config.domains).issubset(cert.names()):
+            continue
         now = pytz.UTC.fromutc(datetime.datetime.utcnow())
+        status = ""
         if cert.is_test_cert:
-            expiration_text = "INVALID: TEST CERT"
-        elif cert.target_expiry <= now:
-            expiration_text = "INVALID: EXPIRED"
+            status = "INVALID: TEST_CERT"
+        if cert.target_expiry <= now:
+            status = status + ",EXPIRED" if status else "INVALID: EXPIRED"
         else:
+            status = checker.ocsp_status(cert.cert, cert.chain, status)
```

Comment:
You're probably right. Of course, I read this right after I finished writing tests for `ocsp_status` :)